### ISSUE TITLE
Audit log: what should Maintainers of projects see?

### ISSUE THREAD

**Issue by brainwane** [pypa/warehouse#5864]
Regarding #5863:

> Each user who maintains at least one project on PyPI will be able to view a log of sensitive actions (performed by ANY user) relevant to projects they act in the Owner Role on.

Just Owners, or also Maintainers? Maybe the Maintainers should be able to see a subset of the things Owners can see?

cc @dstufft @nlhkabu @di 

(This is not urgent right now but will come up as @woodruffw implements the audit log in the future.)

**Comment by trishankatdatadog** [comment]
@brainwane Yes, off the bat I agree that Maintainers should be able to see only a subset of the things Owners can see, but I'm not familiar with all of the things Owners can see, so we maybe we can start there!
